### PR TITLE
feat: add a gradle extraction example for cloud build

### PR DESCRIPTION
**Pull request by danielmoy-google**
docs: documentation for above

fix: fix bug in preprocessor where it linked to javac, not wrapper

### CODE REVIEW

File: kythe/go/extractors/gcp/README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ This also assumes you specify `_BUCKET_NAME` as per the Hello World Test above.
 ```
 gcloud builds submit --config examples/mvn.yaml \
 --substitutions=\
-_BUCKET_NAME="$BUCKET_NAME",\
```
Comment (salguarnieri) [inline kythe/go/extractors/gcp/README.md]: Why remove the quotes?

File: kythe/go/extractors/gcp/examples/mvn.yaml [hunk 2]
```diff
@@ -35,6 +35,6 @@ artifacts:
   objects:
     location: 'gs://${_BUCKET_NAME}/'
     paths:
-      - '/workspace/out/javac-extractor.err'
       - '/workspace/out/test.kzip'
+      - '/workspace/out/javac-extractor.err'
```
Comment (salguarnieri) [inline kythe/go/extractors/gcp/examples/mvn.yaml]: Revert this change unless it is meaningful

File: kythe/go/extractors/gcp/README.md (hunk 1 above)
Comment (danielmoy-google) [reply kythe/go/extractors/gcp/README.md]: they weren't necessary in the first place

File: kythe/go/extractors/gcp/examples/mvn.yaml (hunk 2 above)
Comment (danielmoy-google) [reply kythe/go/extractors/gcp/examples/mvn.yaml]: I do actually want the output object first, since error there will abort attempts to upload the next thing.  The err output is less important to me.

(Ideally in the future we'll figure out how to get both up there in a more resilient fashion, but for now this is better imo)
